profiles/models.py

```diff
@@ -7,6 +7,23 @@
 from jsonfield import JSONField
 
 
+class Employment(models.Model):
+    """
+    A user work_history
+    """
+    city = models.TextField(blank=True, null=True)
```

Comment:
Can you remove `blank=True, null=True` from these fields, except for `state_or_territory`?